System.IO.Abstractions.TestingHelpers.Tests/MockFileDeleteTests.cs

```diff
@@ -35,5 +36,28 @@ public void MockFile_Delete_ShouldThrowArgumentExceptionIfPathContainsOnlyWhites
             // Assert
             Assert.Throws<ArgumentException>(action);
         }
+
+        [Test]
+        public void MockFile_Delete_ShouldThrowDirectoryNotFoundExceptionIfParentFolderAbsent()
+        {
+            var fileSystem = new MockFileSystem();
+            var path = XFS.Path("C:\\test\\somefile.txt");
+
+            Assert.Throws<DirectoryNotFoundException>(() => fileSystem.File.Delete(path));
+        }
+
+        [Test]
+        public void MockFile_Delete_ShouldSilentlyReturnIfNonExistingFileInExistingFolder()
+        {
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>()
+            {
+                { XFS.Path("C:\\temp\\exist.txt"), new MockFileData("foobar") },
+            });
+
+            string filePath = XFS.Path("C:\\temp\\somefile.txt");
+
+            // Delete() returns void, so there is nothing to check here beside absense of an exception
```

Comment:
Consider using `Assert.DoesNotThrow` to support your comment by an actual assertion :)

Reply:
Fixed.